libmscore/types.h

```diff
@@ -300,9 +300,7 @@ enum class GlissandoType {
 //---------------------------------------------------------
 
 enum class GlissandoStyle {
-      ///.\{
-      CHROMATIC, WHITE_KEYS, BLACK_KEYS, DIATONIC
-      ///\}
```

Comment:
I'm not exactly sure what these comments are good for, might be needed for plugins and/or their documentation, @dmitrio95?
Anyway, I'd rather leave them in place, as there's no harm in doing so and the reasons are unclear.

Reply:
Yes, they are exactly for plugins documentation. It looks like Doxygen doesn't want to put enumeration members to the generated documentation if they don't have the corresponding documentation comment. To avoid writing documentation comments for all enumerators which we have (especially given that in most cases such documentation would be not very informative) I made use of an option (should be enabled somewhere in Doxyfile) which allows to define a documentation comment for all entries in a group. These comments do exactly this: they add an empty documentation (`.` symbol) to a group of the enumeration members (group is defined by these braces, `\{` and `\}`). This makes Doxygen add all these entries to the generated documentation, like [here](https://dmitrio95.github.io/plugins/html/namespace_ms.html#aa75b88cb58606dc5b17affc6436eb263).

Reply:
Now I know... interesting